tools/ci-cdk/canary-lambda/write-cargo-toml.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+# Generates a Cargo.toml with the given AWS SDK version for this canary
+
+import argparse
+
+
+def main():
```

Comment:
I wonder if we should just use an actual template for this like https://docs.rs/handlebars/latest/handlebars/

Reply:
I'm inclined to keep it in this Python script just to avoid having to pull dependencies or compile something in order to generate it.